[IA] Relocate Java Automatic pages to new Zero-code section (#4430)
open-telemetry/opentelemetry.io: content/en/docs/zero-code/_index.md

---
title: Zero-code Instrumentation
weight: 2
cSpell:ignore: pageinfo
---

{{% pageinfo color="warning" %}}

🚧 The content of this section is under active development. To track progress,
see [Issue #4427]

[Issue #4427]: https://github.com/open-telemetry/opentelemetry.io/issues/4427

{{% /pageinfo %}}

OpenTelemetry [zero-code instrumentation][] is supported for the languages
listed below in the section index.

If you are using Kubernetes, you can use the [OpenTelemetry Operator for

> Kubernetes][otel-op] to [inject auto-instrumentation libraries][auto] for .NET,

Reviewer: ```suggestion
Kubernetes][otel-op] to [inject zero-code instrumentations][operator-injection] for .NET,
```

> Node.js, Python, Go into your application.

Reviewer 2: ```suggestion
Node.js, Python, or Go into your application.
```

> [auto]: /docs/kubernetes/operator/automatic/

Reviewer: ```suggestion
[operator-injection]: /docs/kubernetes/operator/automatic/
```
